vendor/assets/javascripts/behaviors/selectable_asset.js

```diff
@@ -0,0 +1,17 @@
+Cmsify.SelectableAsset = function(el) {
+  this.$el = $(el);
+  this.select = this.select.bind(this);
```

Comment:
I skipped over this line at first look, I don't this line is currently doing anything, anything set on the prototype should already be assigned and bound to this when the constructor is run. Are you trying to create a class method? If so, you need to do something like this: https://github.com/component/merge-descriptors/blob/master/index.js